quiz.js

```diff
@@ -0,0 +1,65 @@
+// var CarLot = (function (lot) {
+
+//   lot.resetStyles = function (element) {
+
+//   }
+
+//   lot.addStyles = function (element, color) {
+
+//   }
+
+//   return lot
+
+// })(CarLot);
+
+
+CarLot = (function (styles) {
+  console.log(styles)
+  var textInput = document.getElementById("text");
+  var carCard = document.getElementsByClassName("carCard");
+
+  styles.populatePage = function () {
+  var inventory = CarLot.getInventory();
```

Comment:
This variable is not necessary when you have the parameter coming into the function.